apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
```

Comment:
Similar to https://github.com/supabase/supabase/pull/9809/files#r1003825482 let's change the id:

```suggestion
id: rate-limits
```

Reply:
Done